cognite/client/data_classes/datapoints.py

```diff
@@ -980,6 +981,7 @@ def to_pandas(  # type: ignore [override]
             include_aggregate_name (bool): Include aggregate in the column name
             include_granularity_name (bool): Include granularity in the column name (after aggregate if present)
             include_status (bool): Include status code and status symbol as separate columns, if available.
+            include_unit (bool): Include units as a second level in the columns. If availible unit external IDs will be used, if not the basic unit property of the time series will be used. If the time series does not have a unit, the columns will not have a second level.
```

Comment:
I also wonder if we should only support "proper" units as a way of saying "bye bye" to the previous free-text field. Wdyt?